spriter/util/MathUtils.hx

```diff
@@ -120,7 +120,7 @@ class MathUtils
 		var d2:Float;
 		var i:Int;
 
-		// First try a few iterations of Newton's method -- normally very fast.
+		// First try a few iterations of Newton's method -- normally very xml.
```

Comment:
could you write fast instead of xml in the comment please

Reply:
Oops, nice catch. Find and replace bit me :)